lib/rubocop/cop/rspec/repeated_subject_call.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks for repeated calls to subject missing that it is memoized.
+      #
+      # @example
+      #   # bad
+      #   it do
+      #     subject
+      #     expect { subject }.to not_change { A.count }
+      #   end
+      #
+      #   it do
+      #     expect { subject }.to change { A.count }
+      #     expect { subject }.to not_change { A.count }
+      #   end
+      #
+      #   # good
+      #   it do
+      #     expect { my_method }.to change { A.count }
+      #     expect { my_method }.to not_change { A.count }
+      #   end
+      #
+      class RepeatedSubjectCall < Base
+        include TopLevelGroup
+
+        BLOCK_MSG = 'Calls to subject are memoized, this block is misleading'
```

Comment:
By setting this variable name to MSG, it is not necessary to pass it as an argument to `add_offense`.

```suggestion
        MSG = 'Calls to subject are memoized, this block is misleading'
```

Reply:
Updated, thanks!